vip-cache-manager/vip-cache-manager.php

```diff
@@ -84,8 +86,40 @@ public function manual_purge_message() {
 	public function curl_multi( $requests ) {
 		$curl_multi = curl_multi_init();
 
-		foreach ( $requests as $req ) {
-			if ( defined( 'PURGE_SERVER_TYPE' ) && 'mangle' == PURGE_SERVER_TYPE ) {
+		if ( defined( 'PURGE_BATCH_SERVER_URL' ) && defined( 'PURGE_SERVER_TYPE' ) && 'mangle' == PURGE_SERVER_TYPE ) {
```

Comment:
Should we use `===` here vs `==`?

Reply:
Thanks for noticing this, it's fixed in d1828191.